lib/xypn_seo/xypn_scraper/advisor_list.rb

```diff
@@ -0,0 +1,26 @@
+require 'nokogiri'
+require_relative './html_cleaner'
```

Comment:
When you say "extra forward slashes" are you referring to something like this?

`require_relative '../html_cleaner'`

I would suggest  avoiding this or `require_relative`.

The conventional approach is just to use the `require` statement.  And to use it consistently throughout your app. e.g.

```
require 'nokogiri'
require 'xypn_seo/xypn_scraper/html_cleaner'
```